live-examples/css-examples/images/function-repeating-conic-gradient.html

```diff
@@ -0,0 +1,44 @@
+<section id="example-choice-list" class="example-choice-list large">
+
+<div class="example-choice">
+<pre><code class="language-css">background-image: repeating-conic-gradient(red, orange, yellow, green, blue 60deg);</code></pre>
+<button type="button" class="copy hidden" aria-hidden="true">
+    <span class="visually-hidden">Copy to Clipboard</span>
+</button>
+</div>
+
+<div class="example-choice">
+<pre><code class="language-css">background-image: repeating-conic-gradient(
+     from 0.25turn at 50% 30%, 
+     blue 0 0.1turn, lightblue 0.1turn 0.2turn);</code></pre>
+<button type="button" class="copy hidden" aria-hidden="true">
+    <span class="visually-hidden">Copy to Clipboard</span>
+</button>
+</div>
+
+<div class="example-choice">
+<pre><code class="language-css">background-image: repeating-conic-gradient(
+    from 3deg at 25% 25%, 
+    blue 0, lightblue 5deg, blue 10deg, aqua 15deg , blue 20deg);</code></pre>
+<button type="button" class="copy hidden" aria-hidden="true">
+    <span class="visually-hidden">Copy to Clipboard</span>
+</button>
+</div>
+
+<div class="example-choice">
+<pre><code class="language-css">background-image: repeating-conic-gradient(
+	 from 3deg at 25% 25%,
+     red 6deg, orange 6deg 18deg, yellow 18deg 45deg, 
+     green 45deg 110deg, blue 110deg 200deg, purple 200deg);</code></pre>
```

Comment:
I'm not sure why red and purple are given here since they don't seem to appear in the rendered image.